theory/README.md

```diff
@@ -5,13 +5,17 @@
 Find regexes that match the following. (e.g. find a single regex that matches
 both `antelope` and `antelopes`.)
 
+/antelopes?/g
+
 * Single regex that matches either of these:
+/[a-z]/g
```

Comment:
This'll match *any* letter in the alphabet though! We can use a quantifier to specify the number of `s`'s (0 or 1) to match:

```
/antelopes? rocks? out/
```